.eslintrc.cjs

```diff
@@ -4,7 +4,7 @@ module.exports = {
   root: true,
   env: {browser: true, es2020: true},
   extends: ['eslint:recommended', 'plugin:@typescript-eslint/recommended', 'plugin:react-hooks/recommended'],
-  ignorePatterns: ['.eslintrc.cjs', 'examples'],
+  ignorePatterns: ['.eslintrc.cjs', 'examples', 'dist', 'node_modules'],
```

Comment:
I believe eslint [ignores node_modules by default](https://eslint.org/docs/latest/use/configure/ignore#ignoring-files), though it could be dependent on the version.

We definitely should add `dist` though.